### DISCUSSION TITLE
can i re-publish the topics in loaded bag file?

### DISCUSSION THREAD

**Discussion by panda-lord** [foxglove/community#625] [Q&A] [answered]
hi, is there a way to re-publish the (selected) topics in loaded bag file, back to ros nodes?
this could be helpful for simulation/debug code of ros node offline.

just as ros-plotjuggler could do 

![pic](https://user-images.githubusercontent.com/115387801/216259542-d1cb1542-4c26-47d4-afc7-13332c7fb016.png)

**Comment by defunctzombie** [comment, marked as answer]
We do not have this feature.
